Mathlib/MeasureTheory/Measure/Haar/Basic.lean

```diff
@@ -217,7 +215,7 @@ theorem index_union_eq (K₁ K₂ : Compacts G) {V : Set G} (hV : (interior V).N
     simp only [mem_preimage] at h2g₀
     simp only [mem_iUnion]; use g₀; constructor; swap
     · simp only [Finset.mem_filter, h1g₀, true_and]; use g
-      simp only [hg, h2g₀, mem_inter_iff, mem_preimage, and_self_iff]
+      simp [hg, h2g₀]
```

Comment:
This is not using the new simproc, but in line with the new guidelines - so I changed this one as well.